spec/controllers/events_controller_spec.rb

```diff
@@ -53,6 +53,22 @@
     FactoryBot.build(:league, name: nil).attributes
   }
 
+  let(:valid_small_league_attributes) {
+    FactoryBot.build(:league, owner: @user, max_teams: 2).attributes
+  }
+
+  let(:valid_team1_attributes) {
+    FactoryBot.attributes_for(:team)
```

Comment:
I assume the method `attributes_for` also creates an instance of Team by FactoryBot and then returns the attributes? Then `valid_team1_attributes`, `valid_team2_attributes` and `valid_team3_attributes` are instantiated with different values, which is probably the desired behaviour? I am asking because I have only used this method for returning attributes of an already created instance earlier.